### PR TITLE
Updates for Core PS01

### LINKED ISSUE #76: dcterms:contributor@ChangeRequest either has foaf:Person range or not
CM 3.0 CS02 says that `dcterms:contributor` has range `foaf:Person`. In the comment it then says that it might not necessarily be the case. This is wrong. Range is either `foaf:Person` or `Resource` with a note that it MAY be `foaf:Person` in most cases.

RM 2.1 CS02 got it right: http://docs.oasis-open.org/oslc-domains/oslc-rm/v2.1/cs01/part2-requirements-management-vocab/oslc-rm-v2.1-cs01-part2-requirements-management-vocab.html#requirement

### LINKED ISSUE #307: Prepare Core v3.0 PS01
**Preflight**

- [x] Read the [guidelines](https://github.com/oslc-op/oslc-specs/wiki/Publication-guidelines#overall-process)
- [x] **Notify the PGB of the _intent to publish_** ([template](https://github.com/oslc-op/oslc-specs/wiki/Publication-guidelines#pgb-notice-template), ask a PGB member if you are not on the board). The vote can be initiated at least 14 days after this letter has been sent.
- [x] Create a milestone for this stage ([link](https://github.com/oslc-op/oslc-specs/milestones?direction=asc&sort=due_date))
- [x] Add this issue to the newly created milestone and tag it "Kind: EPIC"
- [x] Search for all open issues and add the relevant ones to the milestone

**ReSpec & ShapeChecker**
- [x] CicleCI has Prettier configuration to check spec HTML correctness and formatting
- [x] CicleCI has ShapeChecker configuration to check shapes and vocabs
- [x] Ensure latest ReSpec is used ([link](https://github.com/oasis-tcs/tab-respec/releases))
- [x] Ensure no critical ReSpec errors are present (see the ReSpec button in the top right corner)
- [x] Base URI updated to the OP archive (`docs.oasis-open-projects.org ` and not `docs.oasis-open.org ` unless it's for a _published_ previous stage)
- [x] `specStatus` set to the desired stage (PSD or higher)
- [ ] `publishDate` set to the date of the PGB vote + 7 days (e.g. if you plan to open the vote on Mar 12, set it to Mar 19).
- [x] No ShapeChecker errors in the CircleCI build ([link](https://circleci.com/gh/oslc-op/oslc-specs))

**Editorial**

- [ ] All issues in the milestone are resolved
- [x] RDF files (if any) have the license headers set correctly. NB! Do not put XML comments before the 2nd line; the first line must always be the XML declaration.
- [x] No broken links other than the stage-to-be-published

> **NB!** Check links using https://addons.mozilla.org/en-GB/firefox/addon/find-broken-links/ (http://validator.w3.org/checklink if the PR for the spec did not touch the links since the last WD), fixing all broken links, and fixing those permanent redirects that appear sensible. **Do not** replace links for redirects from 'external' to 'internal' URIs - for example, never replace a link to `open-services.net/ns` with a link to a specific format or version of a vocabulary. Check accessibility using https://achecker.ca/checker/index.php, fixing issues that seem important

**OASIS**

- [ ] Release Draft created (TODO add a hyperlink here to the Release Draft once created, create one [here](https://github.com/oslc-op/oslc-specs/releases))
- [ ] Release ZIP uploaded ([instructions](https://github.com/oslc-op/oslc-specs/wiki/Publication-guidelines#release-zip-generation-steps), TODO add a hyperlink to the Release ZIP once created)
- [ ] PGB vote initiated ([template](https://github.com/oslc-op/oslc-specs/wiki/Publication-guidelines#pgb-ballot-template), TODO add date)
- [ ] **PGB vote passed** (TODO add hyperlink to the announcement email)
- [ ] OASIS publication request filed under [oasis-open-projects/administration](https://github.com/oasis-open-projects/administration/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen) (TODO add link)
- [ ] OASIS feedback addressed
- [ ] The spec is published to the OASIS _staging_ archive
- [ ] OASIS review passed
- [ ] **The spec is published to the OASIS archive**

**After the publication**

- [ ] Tag the commit that was used to produce published snapshots with `%shortname%-v%version%-%STAGE%%REVISION%`.
- [ ] Promote the draft (pre-)release to a (pre-)release. _PSDs are pre-releases._
- [ ] open-services.net redirects resolve to the current published specs/drafts (Andrew)
- [ ] open-services.net blog announcement (spec editor prepares a blog post, see [here](https://github.com/oslc-op/website/wiki/Creating-A-News-Post) and [there](https://github.com/oslc-op/website/wiki/Contributing-Setup), Andrew will do the deployment). The announcement will automatically appear on the forum.
- [ ] LinkedIn announcement (Axel)
- [ ] Twitter announcement (Andrew)

**Kitchensink (for the first-time publication under the Open Project)**

If this is the first PSD of the spec in the open project, there are many possible issues (accumulated from previous OASIS reviews). Please make a quick check to avoid having them and to speed up the review.

- [x] [ReSpec] `latestVersion` updated to point to the one for the OP, not the TC (only for the first PSD)
- [x] [ReSpec] `additionalArtifacts` links are relative
- [x] [OASIS/RM] in "Notices", the copyright year is missing - between "OASIS Open" and the period
- [x] [OASIS/RM] the links to the "Previous stage" should use http instead of https in this case, since the earlier files were installed with http links. When the HTML is retrieved using https, it is unable to load the associated .css file, so the document is poorly presented.
- [x] [OASIS/RM] add HTTPS to the image references
- [ ] [OASIS/RM] in Vocabulary, I would expect that the first two definition should be placed under 2.1.1, since they are Classes rather than Properties. See https://github.com/oasis-tcs/tab-respec/issues/17 (resolved by ReSpec 2.1.10)

### PR DESCRIPTION
**Pull request by jamsden**
Includes splitting vocab and shapes into separate documents and including .ttl files in additional content.

resolves #307; resolves #76

There will be more changes, this is just getting a start for initial review.

### CODE REVIEW

**Comment by jamsden** [comment]
Other than resolving the issues in the milestone, I'm done with all the edits. The conflicts above will need to be addressed.

File: specs/cm/change-mgt-shapes.ttl [hunk 1]
```diff
@@ -331,7 +321,7 @@
         oslc:name                "contributor" ;
         oslc:occurs              oslc:Zero-or-many ;
         oslc:propertyDefinition  dcterms:contributor ;
-        oslc:range               foaf:Person ;
+        oslc:range               oslc:Any ;
```
Comment (berezovskyi) [inline specs/cm/change-mgt-shapes.ttl:324]: What issue no is this change connected to?

**Comment by jamsden** [comment]
> Jim, except for the `oslc:Any` change, everything looks good. Please also resolve merge conflicts before merging.

https://github.com/oslc-op/oslc-specs/issues/76

**Comment by berezovskyi** [comment]
LGTM, please merge once the conflicts are resolved.

**Comment by berezovskyi** [comment]
Let's merge, the prettier check will not break master, RM fixed in another commit.

**Pull request merged**